metabase/metabase
# Convert `pulse.module.css` to CSS modules

### Description

Removes `pulse.module.css` from the global CSS and uses local styles to (hopefully) make the styling for the components more easily understood.

### How to verify

Go to the Question Alerts button (the bell button on the bottom right of the view) and create an alert. Ensure that there are no weird or obvious style failures when using the email and slack widgets.

### Checklist
